pkgs/applications/science/logic/coq/default.nix

```diff
@@ -129,14 +129,14 @@ self = stdenv.mkDerivation {
     ++ optionals buildIde
       (if versionAtLeast "8.10"
        then [ ocamlPackages.lablgtk3-sourceview3 glib gnome.adwaita-icon-theme wrapGAppsHook ]
-       else [ ocamlPackages.lablgtk ]);
+       else [ ocamlPackages.lablgtk ])
+    ++ optional (versionAtLeast "8.14") [ ocamlPackages.dune_2 ];
 
   postPatch = ''
     UNAME=$(type -tp uname)
     RM=$(type -tp rm)
-    substituteInPlace configure --replace "/bin/uname" "$UNAME"
     substituteInPlace tools/beautify-archive --replace "/bin/rm" "$RM"
-    substituteInPlace configure.ml --replace '"md5 -q"' '"md5sum"'
+    substituteInPlace ${if versionAtLeast "8.14" then "tools/configure/" else ""}configure.ml --replace '"md5 -q"' '"md5sum"'
```

Comment:
Are there still references to `md5` in the `configure.ml` of 8.14?

Reply:
Good point, this was removed in 8.7.